.github/workflows/ci.yaml

```diff
@@ -0,0 +1,43 @@
+name: Dashboard
+on:
+  push:
+    paths:
+      - '**'
+  pull_request:
+    paths:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.13'
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip
+          pip install .
+      - name: Lint code
+        run: |
+          pip install flake8
+          flake8 app
```

Comment:
this and other dev dependencies must be in project additional dev requirements, they must be installed by the previous command with specifier `pip install .[dev]`